en/advanced_config/trimming_guide_fixedwing.md

```diff
@@ -0,0 +1,33 @@
+
+- [FW_AIRSPD_TRIM](../advanced_config/parameter_reference.md#FW_AIRSPD_TRIM) is used by the rate controllers to scale their output depending on the measured airspeed. See [Airspeed Scaling](https://dev.px4.io/en/flight_stack/controller_diagrams.html#airspeed-scaling) for more details on this.
+
+- [TRIM_ROLL](../advanced_config/parameter_reference.md#TRIM_ROLL), [TRIM_PITCH](../advanced_config/parameter_reference.md#TRIM_PITCH) and [TRIM_YAW](../advanced_config/parameter_reference.md#TRIM_YAW) applies trim to the control signals - before mixing. For example, if you have two servos for the elevator, `TRIM_PITCH` applies trim to both of them. Use that when your control surfaces are aligned but the aircraft pitches/rolls/yaws up/down/left/right during manual (not stabilized) flight or if the control signal has a constant offset during stabilized flight.
+
+The correct order to set those parameters is:
+
+1. Trim the servos by physically adjusting the linkages lengths if possible and fine tune by trimming the PWM channels (use `PWM_MAIN/AUX_TRIMx`) on the bench to properly set the control surfaces to their theoretical position.
+
+2. Fly in stabilized mode at cruise speed and set the pitch setpoint offset (`FW_PSP_OFF`) to desired angle of attack. The required angle of attack at cruise speed corresponds to the pitch angle the airplane needs to fly at to keep constant altitude during wing-leveled flight. If you are using an airpeed sensor, also set the correct cruise airspeed (`FW_AIRSPD_TRIM`).
+
+3. Look at the actuator controls in the log file (upload it to [Flight Review](https://logs.px4.io) and ckeck the Actuator Controls plot for example) and set the pitch trim (`TRIM_PITCH`). Set that value to the average offset of the pitch signal during wing-leveled flight.
+
+Step 3 can be performed before step 2 if you don't want to have to look at the log or if you feel comfortable flying in manual mode. You can then trim your remote (with the trim switches) and report the values to  `TRIM_PITCH`  (and remove the trims from your transmitter) or ask someone to update  `TRIM_PITCH` directly during flight via telemetry and QGC.
+
+## Advanced trimming
+Given that the downward pitch moment induced by an asymmetric airfoil increases witch airspeed and when the flaps are deployed, the aircraft needs to be re-trimmed according to the current measured airspeed and flaps position. For this purpose, a bilinear curve (centered at the trim airpeed) function of airspeed and a pitch trim increment function of the flaps state can be defined using the following parameters:
+
+- [FW_DTRIM_R/P/Y_VMIN/VMAX](../advanced_config/parameter_reference.md#FW_DTRIM_R_VMIN) is the roll/pitch/yaw trim value added to `TRIM_ROLL/PITCH/YAW` at min/max airspeed.
+
+- [FW_DTRIM_R/P_FLPS](../advanced_config/parameter_reference.md#FW_DTRIM_R_FLPS) is the roll/pitch trim value added to `TRIM_ROLL/PITCH/YAW` when the flaps are deployed.
```

Comment:
@bresch only minor comment here, not sure if it should be explicitly mentioned that the DTRIM for flaps is applied in full only when flaps are deployed to full. So if a user has a flap scale set (say they are only using 50%) - they should not expect the entire DTRIM to be applied, but only 50% of the DTRIM -- this similarly applies to trimming this value in flight, full flaps should ideally be deployed when trying to find this DTRIM.

Reply:
@bresch Did you address/consider/see this comment? If so, please mark resolved.

Reply:
Ignore that, I see you did :-)